.github/workflows/infra-common.yml

```diff
@@ -63,7 +63,7 @@ jobs:
         run: bash ../scripts/lockResourceGroups.sh
           unlock
           "$(terraform workspace show)"
-          "$(terraform output app-name)"
+          "$(terraform output -json app-name | jq -r .)"
```

Comment:
I'm working on the assumption that `jq` is a guaranteed pre-installed dependency on the github image that runs these commands?

Reply:
Yes, it appears to be in the container images we're using.